app/src/lib/stores/app-store.ts

```diff
@@ -6482,8 +6526,27 @@ export class AppStore extends TypedBaseStore<IAppState> {
       return false
     }
 
+    let bannerType: BannerType
+
+    switch (kind) {
+      case MultiCommitOperationKind.Squash:
+        bannerType = BannerType.SquashUndone
+        break
+      case MultiCommitOperationKind.Reorder:
+        bannerType = BannerType.ReorderUndone
+        break
+      case MultiCommitOperationKind.Rebase:
+      case MultiCommitOperationKind.CherryPick:
+      case MultiCommitOperationKind.Merge:
+        throw new Error(
+          `Unexpected multi commit operation kind to undo ${kind}`
+        )
+      default:
+        assertNever(kind, `Unsupported multi operation kind to undo ${kind}`)
+    }
```

Comment:
Again, maybe we can use this for rebase/cherry-picking/merge? Haven't looked into it yet! 😳

Reply:
Yes for cherry-picking.. rebase and merge currently don't support undo.. but maybe they could...